packages/fluentui/docs/src/examples/components/Menu/Usage/MenuExampleTabList.shorthand.tsx

```diff
@@ -7,19 +7,17 @@ const items = [
   { key: 'events', content: 'Upcoming Events' },
 ];
 
-class MenuExampleTabShorthand extends React.Component {
-  render() {
-    return (
-      <Menu
-        defaultActiveIndex={0}
-        items={items}
-        underlined
-        primary
-        accessibility={tabListBehavior}
-        aria-label="Today's events"
-      />
-    );
-  }
-}
+const MenuExampleTabShorthand: React.FC = props => {
+  return (
```

Comment:
_nit_: we don't need `return` statement here

Reply:
Done